packages/chargebee/src/action/index.jsx

```diff
@@ -37,7 +37,9 @@ const Action = ({
       if (!disabled) invoke(onCall, instanceRef.current, global.Chargebee);
     };
 
-    node = <span {...rest} onClick={handleClick} />;
+    const { onStep, ...cleanProps } = rest;
```

Comment:
`onStep` isn't used in this component. where does it come from and why should we omit it here?

Reply:
did you already release the version? this PR has been lying around for quite a while @ozerovs

Reply:
Point is valid, will come back to it once will bump into it again.